src/kudu/consensus/consensus_meta.cc

```diff
@@ -315,14 +342,14 @@ ConsensusMetadata::ConsensusMetadata(FsManager* fs_manager,
     : fs_manager_(CHECK_NOTNULL(fs_manager)),
       tablet_id_(std::move(tablet_id)),
       peer_uuid_(std::move(peer_uuid)),
-      last_pruned_term_(-1),
       has_pending_config_(false),
       flush_count_for_tests_(0),
       on_disk_size_(0) {
   // This is not really required as default values but specifying explicitly
   // since correctness is dependent on it.
-  last_known_leader_.set_uuid("");
-  last_known_leader_.set_election_term(0);
+  pb_.mutable_last_known_leader()->set_uuid("");
+  pb_.mutable_last_known_leader()->set_election_term(0);
+  pb_.set_last_pruned_term(-1);
```

Comment:
does it need a Flush here?

Reply:
I don't think so. The constructor is private and is called in only two places:
1. Load - Already reading from disk
2. Create - Flush argument already is passed.

Let me know if you think I missed something.